pandas/tests/io/test_parquet.py

```diff
@@ -448,9 +457,11 @@ class TestParquetFastParquet(Base):
     def test_basic(self, fp, df_full):
         df = df_full
 
-        # additional supported types for fastparquet
+        # additional supported types for fastparquet>=0.1.4
```

Comment:
you can leave out the ">=0.1.4" as the timedelta is not specific for the newer versions